package.json

```diff
@@ -100,8 +100,11 @@
     "@ionic/angular": "^5.5.2",
     "@nrwl/angular": "11.5.2",
     "@types/prismjs": "1.9.0",
-    "chart.js": "3.3.2",
+    "chart.js": "3.4",
+    "chartjs-adapter-date-fns": "^2.0.0",
     "chartjs-plugin-annotation": "^1.0.2",
+    "chartjs-plugin-crosshair": "github:troelslenda/chartjs-plugin-crosshair#master",
```

Comment:
First of all, gj creating the plugin! 

I need to check up on bankdatas policy regarding  having dependencies like this (if any exists). 

As a minimum I think we should publish it as a package to npm, such that we can lock the version in Kirby. Pulling from your master branch essentially allows you to include arbitrary code in Kirby. Furthermore we also need the guarantee that it will not be unpublished which [npm provides](https://docs.npmjs.com/unpublishing-packages-from-the-registry#how-to-unpublish).

Reply:
To be clear, I did not create the plugin. Then it would in deed be published to npm registry.
This plugin: https://chartjs-plugin-crosshair.netlify.app was simply modified to
1. Work with Typescript
2. Have the option to enabled for certain charts and not all.

package-json.lock states the SHA of master to checkout, so I wouldn't be able to add any extra code unless you actively update the package.

I do not encourage this aproach myself but I'm unsure what else to do.

Reply:
> package-json.lock states the SHA of master to checkout, so I wouldn't be able to add any extra code unless you actively update the package. 

Good point. I will check pu with someone who knows more about this stuff than me, whether it is okay or not 😄

Reply:
chartjs-plugin-marker implemented locally as a substitute for chartjs-plugin-crosshair